build.gradle

```diff
@@ -79,6 +82,21 @@ allprojects {
         configFile file("${project.rootDir}/config/checkstyle.xml")
     }
 
+    detekt {
+        config = files("${project.rootDir}/config/detekt/detekt.yml")
+        autoCorrect = false
+        buildUponDefaultConfig = false
+        disableDefaultRuleSets = false
+        ignoreFailures = false
+        parallel = false
+        debug = false
+        reports {
+            html.enabled = true
```

Comment:
👋 @malinajirka !

Yes, I was thinking about that too. I choose to not disable any of the formats for two reasons:
1) The output files are very small anyway, thus there will not be a build overhead,
1) I don't know how and which format our CI will going to be using in the future, for reporting purposes, so I enabled all possible formats to make it easier for future integration.

PS: We can definitely have only one of those enabled if you like. Let me know which one you prefer. @loremattei

Reply:
I think `xml` is enough for CI as it's the one used by the [plugin we use to post comments on PRs](https://github.com/tomasbjerre/violation-comments-to-github-gradle-plugin). 👍

Reply:
Many thanks for adding to that @loremattei !

So, since `xml` is required by CI, and IMHO `html` is a real nice to have locally, to get a nice representation of the issue and how to fix it, I would have those two enabled. This might leave us with only `txt` to decide to disable, but even that I suggest to keep it enabled since I find myself sometimes looking at this find to figure what the problem more rapidly. I don't think those files are too much overhead and as such I would have the enabled (at least for now).

@loremattei @malinajirka your thoughts?

PS: I will be fine even if we decide to disable all of them and ending up having `xml` the only one being enabled.

Reply:
If it doesn't take any significant amount of time, I agree that keeping all of them is fine.